src/shared/Form/FileUpload.tsx

```diff
@@ -30,39 +30,69 @@ export const FileUpload: React.StatelessComponent<
       accept={'.pdf'}
     >
       {({ getRootProps, getInputProps }) => (
-        <section className={'drop-zone'}>
-          <div {...getRootProps()} className={'container'}>
+        <section className="drop-zone">
+          <div {...getRootProps()} className="container">
             <input {...getInputProps()} />
-            <div className={'drop-content'}>
-              <img src={Upload} alt={'upload'} />
-              <p>Drag and drop or click here to upload</p>
-              <p className={'subtitle'}>
+            <div className="drop-content">
+              <img src={Upload} className="art" alt="upload" />
+              <div className="title">Drag and drop or click here to upload</div>
+              <div className="subtitle">
                 {fileName || 'You can upload a PDF file up to 4MB'}
-              </p>
+              </div>
             </div>
           </div>
           <style jsx>{`
-            .subtitle {
-              color: ${theme.colors.black30};
-            }
             .container {
               text-align: center;
               height: 100%;
             }
+
+            .container:focus {
+              outline: none;
+            }
+            
             .drop-zone {
               width: min(100vw, 960px);
               height: 280px;
-              border: 4px dashed #bcbcbc;
+              border: 4px dashed ${theme.colors.black20};
```

Comment:
Did you mean to change the underlying color? black30 is #bcbcbc

Reply:
Yeah, went a shade lighter on purpose